[
{
"cve_id": "CVE-2024-32982",
"patch_date": [
"2024-05-06 08:37:34+02:00"
],
"patch_files": [
"tests/e2e/test_routing/test_path_resolution.py",
"tests/unit/test_static_files/test_file_serving_resolution.py",
"tests/unit/test_static_files/test_static_files_validation.py",
"litestar/static_files/base.py"
],
"earliest_patch_date": "2024-05-06 08:37:34+02:00",
"earliest_vuln_date": "2022-11-05 13:44:15+01:00",
"vuln_files": [
"tests/unit/test_static_files/test_file_serving_resolution.py",
"tests/unit/test_static_files/test_static_files_validation.py",
"litestar/static_files/base.py"
],
"impact": {},
"cwes": [],
"fix_duplicate_type": false,
"fix_duplicate_types": [],
"intro_duplicate_type": false,
"intro_duplicated_types": [],
"file_fixed_before_fixed_again": false,
"files_fixed_before_fixed_again": [],
"file_fixed_before_vuln_again": false,
"files_fixed_before_vuln_again": []
},
{
"cve_id": "CVE-2024-42370",
"patch_date": [
"2024-08-09 08:48:31-05:00"
],
"patch_files": [
".github/workflows/docs-preview.yml",
".github/workflows/ci.yml"
],
"earliest_patch_date": "2024-08-09 08:48:31-05:00",
"earliest_vuln_date": "2023-05-20 14:42:10+02:00",
"vuln_files": [
".github/workflows/docs-preview.yml",
".github/workflows/ci.yml"
],
"impact": {},
"cwes": [],
"fix_duplicate_type": false,
"fix_duplicate_types": [],
"intro_duplicate_type": false,
"intro_duplicated_types": [],
"file_fixed_before_fixed_again": false,
"files_fixed_before_fixed_again": [],
"file_fixed_before_vuln_again": false,
"files_fixed_before_vuln_again": []
}
]